dropwizard-dependencies/pom.xml

```diff
@@ -87,6 +93,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.glassfish.hk2</groupId>
+                <artifactId>hk2-bom</artifactId>
+                <version>${hk2.version}</version>
```

Comment:
Does it really make sense to declare a specific version of HK2?
If we do, we'll always have to keep in sync with what version Jersey is using to avoid compatibility issues.

Reply:
Yes it will need to stay in sync. We were using hk2 before without being explicit so relying on it being pulled in transitively from other jersey dependencies.

http://maven.apache.org/plugins/maven-dependency-plugin/analyze-only-mojo.html#ignoredUsedUndeclaredDependencies is the only mechanism I know of to keep using hk2 as we were before. This would essentially be disabling maven-dependency-plugin for certain artifact coordinates.